src/lightning_app/core/flow.py

```diff
@@ -763,3 +763,15 @@ def load_state_dict(self, flow_state, children_states, strict) -> None:
                 child.set_state(state)
             elif strict:
                 raise ValueError(f"The component {child_name} wasn't instantiated for the component {self.name}")
+
+
+class _RootFlow(LightningFlow):
+    def __init__(self, work):
+        super().__init__()
+        self.work = work
+
+    def run(self):
+        self.work.run()
+
+    def configure_layout(self):
+        return [{"name": "Main", "content": self.work}]
```

Comment:
```suggestion
        return [{"name": "Main", "content": self.work.url}]
```

Reply:
or does the `content` fetch that automatically?

Reply:
Yes, it gets the URL directly ;)  You can pass the url only if you want to add more than the root path.